src/components/LoanCards/ExpandableLoanCard/ExpandableLoanCardExpanded.vue

```diff
@@ -56,24 +76,28 @@ export default {
 		};
 	},
 	methods: {
-		collapseCard() {
+		collapseCardAndPauseHover(timeToPause) {
 			this.clearHoverLoan();
-			this.pauseHover(500);
+			this.pauseHover(timeToPause);
+		},
+		activeLoanWillChange() {
+			this.expanded = false;
+			this.show = false;
```

Comment:
I like this, it's another lifecycle hook into how the data is updated that just cleaned up the animation in cases where the user hovers over a loan that can't expand